requirements.txt

```diff
@@ -1,4 +1,4 @@
-numpy==1.8.0
-numpydoc==0.4
-networkx==1.8.1
+numpy>=1.8.0
+numpydoc>=0.4
+networkx>=1.8.1
```

Comment:
My god! For a second there I thought this is why the add-on was uninstallable in certain situations ...

This file isn't used. Please remove it.